src/content/en/updates/2016/10/navigator.share.md

```diff
@@ -0,0 +1,126 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Sharing is caring. Chrome is running an Origin Trial to enable
+             native sharing on the web.
+
+{# wf_published_on: 2016-10-19 #}
+{# wf_updated_on: 2016-10-19 #}
+
+Good news, everybody! [Matt Giuca](https://twitter.com/mgiuca) on the Chrome
+team has been working on a [simple
+API](https://github.com/WICG/web-share/blob/master/docs/interface.md) called
+[Web Share](https://github.com/WICG/web-share/blob/master/docs/explainer.md)
+that allows websites to invoke the native sharing capabilities of the host
+platform.
+
+There have been a number of ways to invoke native sharing capabilities on the
+platform, but they all have significant drawbacks.  There was [Web
+Intents](https://en.wikipedia.org/wiki/Paul_Kinlan) (dead), there is protocol
+handling (poor support on mobile), there is direct sharing to a well-known
+service URL such as Twitter's, and there is also the [Android intent: URL
+syntax](https://paul.kinlan.me/sharing-natively-on-android-from-the-web/) (well,
+Android-only, and Apps had to opt-in).
+
+The Web Share API is important because it gives the user control of how and
+where the data is shared.
+
+In Chrome 55 (Beta as of October 2016), we've enabled an [Origin
+Trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-guide.md)
+that lets you integrate the Web Share API into your site. The origin trial means
+this API is not generally available to all sites; instead you need to register
+to get access during this Trial phase. Over this time, the API will likely
+change and break in unexpected ways and we are looking for as much feedback as
+possible.
+
+The Web Share API is
+[promise](https://developers.google.com/web/fundamentals/getting-started/primers/promises)-based
+single method API that takes an object with properties title, text and url.
+
+    navigator.share({
+        title: document.title,
+        text: window.location.href,
+        url: window.location.href
+    }).then(() => console.log('Successful share'))
+    .catch(() => console.log('Error sharing:', error));
+
+Once invoked it will bring up the native picker (see video) and allow you to
+share the data with the app that the user prefers.
+
+<div class="video-wrapper">
```

Comment:
I'd put the video first, then the text (once invoked...), and add a `<div class="clearfix"></div>` after the text. That way you'll get text on the left, video on the right and the next chuck of text will flow nicely below.

```
<div class="video-wrapper">
  <iframe class="devsite-embedded-youtube-video" data-video-id="lhUzYxCvWew"
          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
  </iframe>
</div>

Once invoked it will bring up the native picker (see video) and allow you to
 share the data with the app that the user prefers.
<div class="clearfix"></div>
```